src/EditorFeatures/CSharp/SplitStringLiteral/SplitStringLiteralCommandHandler.StringSplitter.cs

```diff
@@ -10,18 +10,15 @@
 namespace Microsoft.CodeAnalysis.Editor.CSharp.SplitStringLiteral
 {
     using Microsoft.CodeAnalysis.Indentation;
+    using Microsoft.CodeAnalysis.Options;
 
     internal partial class SplitStringLiteralCommandHandler
     {
         private abstract class StringSplitter
         {
             protected static readonly SyntaxAnnotation RightNodeAnnotation = new SyntaxAnnotation();
 
-            protected static readonly SyntaxToken PlusNewLineToken = SyntaxFactory.Token(
-                leading: default,
-                SyntaxKind.PlusToken,
-                SyntaxFactory.TriviaList(SyntaxFactory.ElasticCarriageReturnLineFeed));
```

Comment:
💭 I would expect an elastic trivia to get updated to have the correct form, without needing to separately track the newline configuration inside the fix. I wonder if we can get the `CodeAction` infrastructure to do this for us.